storage/aptosdb/src/state_store/mod.rs

```diff
@@ -50,20 +50,14 @@ pub(crate) struct StateStore {
 // upcasting coercion for now. Should change it to a different trait once upcasting is stablized.
 // ref: https://github.com/rust-lang/rust/issues/65991
 impl DbReader for StateStore {
-    /// Get the state value with proof given the state key and version
-    fn get_state_value_with_proof_by_version(
+    /// Returns the latest state snapshot strictly before `next_version` if any.
+    fn get_state_snapshot_before(
```

Comment:
why do you swap the order of two functions in this file?

Reply:
clion asked me to do that to follow the order in the trait.. How do you find it, haha